www/src/pages/blog.astro

```diff
@@ -24,29 +22,40 @@ const formattedPosts = sortedPosts.map((post) => {
   };
   return formattedPost;
 });
-
-const getAuthorSlug = (author: string) => {
-  return kebabify(author);
-};
 ---
 
-<Layout title={title}>
-  <Stack as='ul' gap={12} axis='y' class='u-list-style:none'>
+<Base title={title}>
+  <Stack as="ul" gap={12} axis="y" class="blog-list u-list-style:none">
     {
       formattedPosts.map((post) => (
         <li>
-          <Stack as='article' gap={2}>
-            <Text as='h3' variant='heading3'>
-              <a href={post.url} aria-label={`Read ${post.title}`}>
-                {post.title}
-              </a>
-            </Text>
-            <Text as='p' variant='small'>
+          <Stack as="article" gap={2} class="article">
+            <a
+              href={post.url}
+              aria-label={`Read ${post.title}`}
```

Comment:
Also removed this aria-label override. Adding additional words for screen readers when there’s no visual indicator isn’t necessary. Screen reader users should just have the links read the same as sighted users. This creates unnecessary repetition.